website/docs/concepts/canbus_service/README.md

```diff
@@ -5,4 +5,37 @@ title: CANBUS
 
 # Canbus Service Overview
 
-(Coming soon!)
+The Canbus service is an integral component of the Amiga platform.
+This service offers direct control over the robot's motors, allowing users to both
+publish motor states and send specific commands to manipulate the robot's movements.
+
+## Overview
+
+- **Purpose**: Control and manage the motors on the Amiga, handling command reception,
+processing, and motor instruction.
+- **Communication**: Direct interface with motors via Canbus, a communication standard
+or microcontrollers and devices.
+- **Dependencies**: Standalone service, but pivotal for many autonomous operations.
+Notably, the Controller service is heavily reliant on the Canbus service.
+
+## API
+
+1. `/twist`: Receives twist commands (linear and angular velocities) and instructs the
+motors accordingly.
+2. `/can_message`: Accepts a specific Protobuf message format (details TBD).
+3. `/get_battery_state`: Returns the battery state.
+A float value indicating the battery's state of charge, ranging [0.0, 1.0].
+If no motors are connected, it returns -1.0.
+
```

Comment:
These are only the `RequestReply` API. The streams are:
- `/twist` (`canbus_pb2.Twist2d`)
- `/raw_messages` (`canbus_pb2.RawCanbusMessages`)
- `/state` (`amiga_v6_pb2.AmigaV6CanbusState`)
- `/motor_states` (`canbus_pb2.MotorStates`)